### PR TITLE
Issue42 comments lesson 2

### LINKED ISSUE #42: Some comments on lesson 2 "Introduction to agile"
**Style**

- The word "agile" should be replaced with "Agile" everywhere. 
- The first question is `What is ‘agile’?` and the lesson starts with `Waterfall model`. Can we start with an introduction about Agile instead?

**Text:**
The lesson is clear about the difference between "Agile" and "Waterfall model" and lists the benefits and downsides of Agile. However, it is not clear `Why should I work in an agile way?` which is the second question of the lesson. Let's add a paragraph where we focus on the advantages of Agile while working in a team and explain how an Agile way helps to increase the team benefits (mentioned in the previous lesson), in other words, let's make a link between lessons 1 and 2 here. Or perhaps we can rephrase the question.

### PR DESCRIPTION
**Pull request by raar1**
Fixes #42

In response to Sarah's comments on lesson 2:

>    The word "agile" should be replaced with "Agile" everywhere.

Done.

>    The first question is What is ‘agile’? and the lesson starts with Waterfall model. Can we start with an introduction about Agile instead?

I agree that it is weird to jump straight in to 'Waterfall' in a lesson titled 'Agile', so I have added a brief introduction stating what Agile refers to, in general terms. However, I do think the easiest way to make people understand what Agile is is to describe why it exists, what it is meant to be a reaction to. So I still put the Waterfall section before the more detailed section on Agile, but there is now text explaining what we're doing, so hopefully it doesn't seem like a weird order anymore.

> The lesson is clear about the difference between "Agile" and "Waterfall model" and lists the benefits and downsides of Agile. However, it is not clear Why should I work in an agile way? which is the second question of the lesson. Let's add a paragraph where we focus on the advantages of Agile while working in a team and explain how an Agile way helps to increase the team benefits (mentioned in the previous lesson), in other words, let's make a link between lessons 1 and 2 here. Or perhaps we can rephrase the question.

I've added a paragraph to the end of the 'Solutions' section that specifically addresses this question. I've tried to relate some of the benefits of Agile to benefits of teams from lesson 01, but much of it is orthogonal. Please feel free to suggest improvements here... :)

### CODE REVIEW

File: _episodes/02-introduction-to-agile.md [hunk 1]
```diff
@@ -40,10 +46,10 @@ Learn more about the waterfall model [here](https://www.tutorialspoint.com/sdlc/
 
 # Agile manifesto
 The Agile way of working contrasts the Waterfall model and mitigates most of its drawbacks.
-Watch [this video introducing the agile manifesto](http://growingagile.thinkific.com/courses/take/scrumbasics/lessons/967803-agile-manifesto-lesson).
+Watch [this video introducing the Agile manifesto](http://growingagile.thinkific.com/courses/take/scrumbasics/lessons/967803-agile-manifesto-lesson).
```
Comment (raar1) [inline _episodes/02-introduction-to-agile.md:49]: @SarahAlidoost Is it normal that this link to the video takes you to a page where you need an account to log in?

Comment (svenvanderburg) [reply _episodes/02-introduction-to-agile.md:49]: Yes this is normal. We mention making an account in the setup instructions.

File: _episodes/02-introduction-to-agile.md [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
 - "List benefits of following Agile manifesto"
 keypoints:
-- "The agile manifesto provides good pointers for successfully working on software projects as a team"
+- "The Agile manifesto provides good pointers for successfully working on software projects as a team"
 ---
+
+# What is Agile?
+The term 'Agile' refers to a broad family of software development methodologies. The core concepts linking these methodologies is iterative development, with frequent interaction between interested parties to decide and update requirements.
+
+Is that all? Well, there is a bit more to it than this, but the best way to get a feel for Agile is to understand why it exists in the first place: it is a reaction to the type of 'Waterfall' methodology that was prevalent at the time. Let us first look at what 'Waterfall' means, to see the problems that Agile is trying to solve.
```
Comment (svenvanderburg) [inline _episodes/02-introduction-to-agile.md:19]: Really like this addition :)

File: _episodes/02-introduction-to-agile.md [hunk 3]
```diff
@@ -79,15 +85,18 @@ Watch [this video introducing the agile manifesto](http://growingagile.thinkific
 > > - Because of incremental nature it is hard to say when something is done.
 > > - You need a high level of collaboration that is hard to maintain.
 > > - You need support from the management to do it effectively.
+> >
+> > ### Summary: Why should I work in an Agile way?
+> > Essentially, Agile works well for projects in which we do not know all the requirements in detail at the start, and in which requirements are likely to change during the course of the project. This is very likely the case in a research project, which naturally have many unknowns and the requirements of software can change as new information is collected. Both Agile and Waterfall are suitable for teams, but an Agile approach gives the team more power to adapt and organize itself.
```
Comment (svenvanderburg) [inline _episodes/02-introduction-to-agile.md:90]:
```suggestion
> > Essentially, Agile works well for projects in which we do not know all the requirements in detail at the start, and in which requirements are likely to change during the course of the project. This is very likely the case in a research project, which naturally have many unknowns and the requirements of software can change as new information is collected. Both Agile and Waterfall are suitable for teams, but an Agile approach gives the team more power to adapt and organize itself.

In short: following the agile principles helps teams to make the benefits of working in a team outweigh the downsides. 
```

Comment (svenvanderburg) [reply _episodes/02-introduction-to-agile.md:90]: Maybe add this to link it to the first episode more?

**Review by svenvanderburg — APPROVED** [review]
Really like your additions! I have added one small suggestion.

File: _episodes/02-introduction-to-agile.md (hunk 1 above)
Comment (raar1) [reply _episodes/02-introduction-to-agile.md:49]: Ah yes I hadn't seen that section

File: _episodes/02-introduction-to-agile.md (hunk 2 above)
Comment (raar1) [reply _episodes/02-introduction-to-agile.md:19]: :)

**Pull request merged**